pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/codestyle/CommentDefaultAccessModifierRule.java

```diff
@@ -57,8 +57,9 @@ protected Collection<String> defaultSuppressionAnnotations() {
         Collection<String> ignoredStrings = new ArrayList<>();
         ignoredStrings.add("com.google.common.annotations.VisibleForTesting");
         ignoredStrings.add("android.support.annotation.VisibleForTesting");
+        ignoredStrings.add("co.elastic.clients.util.VisibleForTesting");
         ignoredStrings.add("org.junit.jupiter.api.Test");
-        ignoredStrings.add("org.junit.jupiter.api.ParameterizedTest");
+        ignoredStrings.add("org.junit.jupiter.api.1");
```

Comment:
This change seems to be unintended - please undo.

Reply:
Yes unintended.  Error has been corrected.